### PR TITLE
(LEARNVM-509) Add Web Terminal to GitBook

### PR DESCRIPTION
**Pull request by marrero984**
LEARNVM-509 #resolved #time 4h

Added a layout directory and html page to use Gitbook's theming
 options, and add an abalone terminal to the right of the page.
 Any window smaller than full-size on a MBP will switch to vertical
 styling, otherwise it cuts off too much of the terminal when first
 loaded.

### CODE REVIEW

**Comment by marrero984** [comment]
screen shot 2017-06-06 at 1 15 16 pm

File: _layouts/website/page.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% endblock %}
```
Comment (kjhenner) [inline _layouts/website/page.html]: Is this endblock tag an error?

Comment (marrero984) [inline _layouts/website/page.html]: Nope.  Part of the theming syntax.

https://toolchain.gitbook.com/themes/

Comment (kjhenner) [inline _layouts/website/page.html]: I get an error when I try to build with that {% endblock %} at the beginning of the file. It seems to work correctly when I delete it.

**Comment by kjhenner** [comment]
Is there some way we could get rid of the extra scroll-bar on the right side? It looks like there's a scroll bar for the terminal itself as well as the element that contains it. Actually, if we could get rid of both, that would be ideal, as scrolling back doesn't work correctly with tmux anyway.
screen shot 2017-06-06 at 1 50 32 pm

**Comment by kjhenner** [comment]
Some of the splash-page text in the terminal gets cut off when that element is less than 700px.
screen shot 2017-06-06 at 2 04 02 pm

**Pull request closed without merging**